### PR TITLE
[RFR] Autofill access based on privatePrefix

### PR DESCRIPTION
**Pull request by FWeinb**
See #340

### CODE REVIEW

File: src/annotation/annotations/access.js [hunk 1]
```diff
@@ -6,6 +9,21 @@ export default function access() {
       return text.trim();
     },
 
+    autofill(item) {
+      if (env.privatePrefix === false) { return; }
```
Comment (HugoGiraudel) [inline src/annotation/annotations/access.js]: Missing linebreaks here.

**Comment by HugoGiraudel** [comment]
Looks alright to me. We should not forget to PR the docs.

**Comment by HugoGiraudel** [comment]
Docs here: https://github.com/SassDoc/sassdoc.github.io/pull/110.

**Comment by HugoGiraudel** [comment]
LGTM. :+1:

**Comment by HugoGiraudel** [comment]
@FWeinb Whenever you feel like it.

**Comment by pascalduez** [comment]
Manually merged.

**Pull request closed without merging**